src/package.json

```diff
@@ -32,7 +33,7 @@
     "test:unit": "npm run compile && npm run unit",
     "lint:fix": "eslint --fix --output-file ./reports/eslint/report.html --format html -c .eslintrc.js --ext .ts .",
     "compile": "./node_modules/.bin/tsc",
-    "lint": "eslint **/**.ts **/**.test.ts",
+    "lint": "eslint --ext .ts .",
```

Comment:
Stelios was talking about adding prettier to the FE and I wondered wether it's worth doing the same for BE? https://github.com/govuk-one-login/ipv-cri-f2f-api/pull/292/files#diff-cc5a2f170768969f124108ad3be7fdbcbed774c11774d99a87a3a78fef4ab97bR35

Reply:
Yeah sounds good, conscious that it will probably increase the diff are you happy for us to do it in another ticket?

Reply:
Yea for sure let's do in diff ticket